pkg/virt-launcher/virtwrap/util/libvirt_helper.go

```diff
@@ -408,3 +416,18 @@ func SetupLibvirt() error {
 
 	return nil
 }
+
+func convertToLibvirtXML(domspec string) (string, error) {
+
+	dom := &libvirtxml.Domain{}
+	err := xml.Unmarshal([]byte(domspec), &dom)
+	if err != nil {
+		return "", err
```

Comment:
please do a `fmt.Errorf("failed unmarshaling libvirt xml: %v", err)` so it's easier to find where it's comming from